charts/temporal/values.yaml

```diff
@@ -242,6 +242,33 @@ server:
     containerSecurityContext: {}
     topologySpreadConstraints: []
     podDisruptionBudget: {}
+  internal-frontend:
```

Comment:
seems to me the helm chart is uses `camelCase` convention, would it ok if we renamed `internal-frontend` to `internalFrontend`? It would also makes the usage a bit easier, we could use `{{ if $.Values.server.internalFrontend.enabled }}` directly.

Reply:
@adamko147 I had initially done this as `internalFrontend` but recall these identifiers are actually used to generate the kubernetes services and it doesn't allow uppercase letters - hence `internal-frontend` which corresponds to what the actual temporal feature is named.

Reply:
huh, thanks, did not realize that.

Reply:
Just as a proposition: you can create a simple helper to use internalFrontend as a "yaml" part and internal-frontend in names when needed:

```
{{- define "serviceName" -}}
    {{- $service := index . 0 -}}
    {{- if eq $service "internalFrontend" }}
        {{- print "internal-frontend" }}
    {{- else }}
        {{- print $service }}
    {{- end }}
{{- end -}}
```

Usage:

```
{{ $service := include "serviceName" (list "frontend" "internalFrontend" "matching" "history" "worker") }}
```